collector/index.md

```diff
@@ -0,0 +1,425 @@
+databases on the same server, one is considered the primary. This is where helper functions
+are expected to be defined, and the one we connect to for server-wide metrics.
+
+<table>
+  <thead>
+    <tr>
+      <th>setting</th>
+      <th>default</th>
+      <th>description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>db_url (DB_URL)</td>
+      <td>n/a - either this or individual db_* settings are required</td>
+      <td>URL of the database server to monitor</td>
+    </tr>
+    <tr>
+      <td>db_name (DB_NAME)</td>
+      <td>n/a - either this or db_url is required</td>
+      <td>name of database to monitor; or, comma-separated list of all databases to monitor,
+      starting with primary (last entry can be <code>*</code> to monitor all databases on server)</td>
+    </tr>
+    <tr>
+      <td>db_username (DB_USERNAME)</td>
+      <td>n/a - either this or db_url is required</td>
+      <td>user to connect as</td>
+    </tr>
+    <tr>
+      <td>db_password (DB_PASSWORD)</td>
+      <td>n/a - either this or db_url is required</td>
+      <td>password to connect with</td>
+    </tr>
+    <tr>
+      <td>db_host (DB_HOST)</td>
+      <td>n/a - either this or db_url is required</td>
+      <td>host to connect to</td>
+    </tr>
+    <tr>
+      <td>db_port (DB_PORT)</td>
+      <td>n/a - either this or db_url is required</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>db_sslmode (DB_SSLMODE)</td>
+      <td>prefer</td>
+      <td>sslmode to connect with (see <a target="_blank" href="https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-SSLMODE">Postgres documentation</a> for more details)</td>
+    </tr>
+    <tr>
+      <td>db_sslrootcert (DB_SSLROOTCERT)</td>
+      <td>system certificate store (?)</td>
+      <td>path to SSL certificate authority (CA) certificate(s) to use to verify the server's certificate, or one of <code>rds-ca-2015-root</code> or <code>rds-ca-2019-root</code> to use the built-in AWS RDS certificates</td>
+    </tr>
+    <tr>
+      <td>db_sslrootcert_contents (DB_SSLROOTCERT_CONTENTS)</td>
+      <td>n/a, see above</td>
+      <td>as above, but actual contents of the certificate(s)</td>
+    </tr>
+    <tr>
+      <td>db_sslcert (DB_SSLCERT)</td>
+      <td>[none]</td>
+      <td>file name of the client SSL certificate</td>
+    </tr>
+    <tr>
+      <td>db_sslcert_contents (DB_SSLCERT_CONTENTS)</td>
+      <td>[none]</td>
+      <td>as above, but actual contents of the certificate</td>
+    </tr>
+    <tr>
+      <td>db_sslkey (DB_SSLKEY)</td>
+      <td>[none]</td>
+      <td>location for the secret key used for the client certificate</td>
+    </tr>
+    <tr>
+      <td>db_sslkey_contents (DB_SSLKEY_CONTENTS)</td>
+      <td>[none]</td>
+      <td>as above, but actual contents of the key</td>
+    </tr>
+  </tbody>
+</table>
+
+## AWS settings
+
+Only relevant if you are running your database in AWS RDS. Note that the <code>aws_endpoint_*</code>
```

Comment:
If you use the built-in markdown tags this gets escaped correctly (backticks), otherwise you need to prefix underscores like this: `\_`